cmd/aries-js-worker/go.mod

```diff
@@ -8,10 +8,9 @@ go 1.16
 
 require (
 	github.com/google/uuid v1.1.2
-	github.com/hyperledger/aries-framework-go v0.1.7-0.20210409151411-eeeb8508bd87
+	github.com/hyperledger/aries-framework-go v0.1.7-0.20210422133815-2ef2d99cb692
```

Comment:
and here

Reply:
Actually, these imports aren't in go.sum to begin with. I assume this is due to the replace.